src/features/event_episodes/readme.md

```diff
@@ -0,0 +1,45 @@
+# Event Episodes
+
+## Introduction:
+
+This feature allows the user to view the episodes of a disaster on a timeline, select a specific episode or cluster of episodes, and view data related to the selected episode.
+
+## Components
+
+### EpisodesTimelinePanel
+
+A panel containing the timeline of episodes. The user can open and close the panel. If the app is running on a mobile device, the panel opens in a modal window.
+
+### EpisodesTimeline
+
+A component that displays the timeline of episodes. It receives a list of episodes as `episodes` prop and displays them on the timeline. The user can select a specific episode or cluster of episodes to view it on the map. The component also supports data-driven styles for episodes and clusters of episodes.
+
+## Atoms
+
+### episodesResource
+
+An atom that stores a list of episodes associated with the selected disaster. The atom loads data from the server and provides an interface for getting this data. `episodesResource` depends on `currentEventAtom` trough the `episodesResourceDependencyAtom`.
```

Comment:
change _trough_ to _**through**_

Reply:
more details on `currentEventAtom` and `episodesResourceDependencyAtom`. What is inherited from what and what just depends on each other